data/yam/autoyast/create_hdd_gnome_x86_64_raid_encrypted_home.xml

```diff
@@ -0,0 +1,411 @@
+<?xml version="1.0"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <add-on t="map">
+    <add_on_products t="list">
+      <listentry t="map">
+        <media_url>https://updates.suse.com/SUSE/Products/SLE-Module-Basesystem/15-SP5/x86_64/product?uqBLT_zaCrKaPl66l5TOK5kz1Cm-wowzT03E-BxbbNN61C48hWiJ1-7NbkMAlNs-qLDTweuinZKLKOYb-KmIOI3X8uHyFpDJxBf_VePKxTpWdGbNYcOUum6IP-OykxLq-MU_z3vNOIl_Hephpy_xvA9myis34Y5X</media_url>
+        <product>SLE-Module-Basesystem15-SP5-Pool</product>
+        <product_dir/>
+      </listentry>
+      <listentry t="map">
+        <media_url>https://updates.suse.com/SUSE/Products/SLE-Module-Server-Applications/15-SP5/x86_64/product?qKA5x9nh2ee-G2X0H6XSIVLXdT8cH8US1qQJyo9D1eOsLMBqNpY1R7-Qj_5BDQCidvaENJM9lKo7kKVXJm6QUyvdcxYrzLvAB-f-FUSjEFIF0aoWptANPknd0DIc5BrszSXm_foNwr3Q5n_Q-YxzvLcwFtax-XYMfia6jWrvUWK5</media_url>
+        <product>SLE-Module-Server-Applications15-SP5-Pool</product>
+        <product_dir/>
+      </listentry>
+      <listentry t="map">
+        <media_url>https://updates.suse.com/SUSE/Products/SLE-Module-Python3/15-SP5/x86_64/product?4SZ0rt6mKcY6w2TlOlBq3LY4mqjxfU0YIDqjk9z8DB4h0EKIdhBCiqWLRFUHSMFNCaYTGvzCdejnr2zOGCEdN_wgZcZOMmE3pnF04GE6MXUI16Jb7WmLKKne4XcEPvhvYOQqyA1Xk2f_lYXMB4yh7mkJWyzj</media_url>
+        <product>SLE-Module-Python3-15-SP5-Pool</product>
+        <product_dir/>
+      </listentry>
+    </add_on_products>
+  </add-on>
+  <bootloader t="map">
+    <global>
+      <timeout config:type="integer">-1</timeout>
+    </global>
+  </bootloader>
+  <general t="map">
+    <mode t="map">
+      <confirm t="boolean">false</confirm>
+    </mode>
+  </general>
+  <networking t="map">
```

Comment:
what about networking or ntp-client, also looks quite default, they might be auto-generated...

Reply:
We can remove ntp-client, but it seems networking is needed or it will failed to connecting register server. https://openqa.suse.de/tests/14743935#step/installation/6

Reply:
Please check again autoyast_mini, this might work:
```
<networking>
        <keep_install_network config:type="boolean">true</keep_install_network>
</networking>
```

Reply:
Updated.